docs/topics/assembly_building-and-deploying-the-booster-to-openshiftlocal.adoc

```diff
@@ -13,7 +13,8 @@ You have two options for executing the {parameter-mission-name} booster locally
 * xref:deploying-the-booster-using-launcher-osl_{context}[Using {launcher}]
 * xref:deploying-the-{parameter-mission}-booster-using-the-oc-cli-client_{context}[Using the `oc` CLI client]
 
-There is no functional difference between these methods, choose the one you prefer.
+While both methods use the same `oc` commands to deploy your application, using
+{launcher} provides and automated booster deployment workflow that executes the `oc` commands behind the scenes.
```

Comment:
* Replace _and_ with _an_.
* Suggestion: _behind the scenes_ seems a little too metaphorical for me, I'd opt for _for you_ or some such.